flow/FileTraceLogWriter.cpp

```diff
@@ -117,6 +117,7 @@ void FileTraceLogWriter::open() {
 		}
 		else {
 			fprintf(stderr, "ERROR: could not create trace log file `%s' (%d: %s)\n", finalname.c_str(), errno, strerror(errno));
+			issues->addIssue("trace_log_writer_could_not_create_trace_log_file");
```

Comment:
I think the `trace_log_writer` part of this arguably is an implementation detail, and it's probably ok to just remove it:

```suggestion
			issues->addIssue("could_not_create_trace_log_file");
```

Reply:
The intent is that the `trace_log_writer` prefix can server as a poor man's issue type. So later the issues on CC side can be filtered by tools.

Reply:
Could `trace_log` be the prefix? I think this is still pretty clear to a user seeing it in fdbcli, so if it solves your problem too it could be a win-win:

```suggestion
			issues->addIssue("trace_log_could_not_create_file");
```

Reply:
Sure.